.github/workflows/release.yml

```diff
@@ -1,41 +1,15 @@
-name: Release
+name: Release tag pushes on Github
+
 on:
-  workflow_run:
-    workflows: [ "Developer checks" ]
-    types: [ completed ]
+  push:
+    tags:
+      - v**
+
 jobs:
-  # This 'check_tag' job is a hack because Github doesn't provide the full ref from the
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->

👏  **praise** 
Good think to remove this hack